src/components/core/DpEditor/DpBoilerPlate.vue

```diff
@@ -45,6 +45,7 @@
 <script>
 import { CleanHtml } from '../../../directives/CleanHtml/CleanHtml'
 import DpMultiselect from '../form/DpMultiselect'
+import { Tooltip } from '../../../directives'
```

Comment:
We should use one import statement for CleanHtml and Tooltip:
`import { CleanHtml, Tooltip } from '../../../directives'`